Source/rclUE/Public/rclcUtilities.h

```diff
@@ -32,12 +32,11 @@ DECLARE_LOG_CATEGORY_EXTERN(LogROS2Action, Log, All);
         if ((temp_rc != RCL_RET_OK))                                                \
         {                                                                           \
             UE_LOG(LogTemp,                                                         \
-                   Error,                                                           \
+                   Warning,                                                           \
                    TEXT("Failed status on line %d (function %s): %d. Continuing."), \
                    __LINE__,                                                        \
                    *FString(__FUNCTION__),                                          \
                    (int)temp_rc);                                                   \
```

Comment:
The `check `basically is to verify that ROS2 apis return properly, which could only be assured upon LD_LIBRARY_PATH is appended with ROS2 .so lib paths, provided in rclUE plugin.
I guess If not running through run_editor.sh, it should be configured in the IDE for its debugging to work.